OpenRA.Mods.Common/Activities/Resupply.cs

```diff
@@ -148,16 +159,16 @@ public override bool Tick(Actor self)
 
 			if (activeResupplyTypes == 0)
 			{
+				// Resupply properly finished, so ReturnToBase will handle the TakeOffOnResupply
```

Comment:
`TakeOffOnResupply` is now handled in two different places. It might be better to remove the `TakeOff` handling from `ReturnToBase` and handle it here. I moved the `TakeOff` to `ReturnToBase` to handle the `alwaysLand` case, but that can be handled by passing the boolean down to `Resupply` instead (per the above comment).